RecoParticleFlow/PFProducer/src/PFAlgo.cc

```diff
@@ -171,18 +166,16 @@ void PFAlgo::reconstructParticles(const reco::PFBlockHandle& blockHandle, PFEGam
     }
   }  //loop blocks
 
-  if (debug_) {
-    cout << "# Ecal blocks: " << ecalBlockRefs.size() << ", # Hcal blocks: " << hcalBlockRefs.size()
-         << ", # HO blocks: " << hoBlockRefs.size() << ", # Other blocks: " << otherBlockRefs.size() << endl;
-  }
+  LogDebug("PFAlgo::reconstructParticles")
```

Comment:
were you able to select on categories with ```::``` in the message logger?
There are some special characters that can not be a part of the category and I'm not sure if colons are among them or now.

Reply:
In the end I turned PFAlgo on or off just through particleFlowTmp. I used `grep` later to filter on a specific function.